src/plugins/intel_gpu/include/intel_gpu/primitives/border.hpp

```diff
@@ -69,7 +72,9 @@ struct border : public primitive_base<border> {
     /// @brief Type of border that needs to be added to the input.
     ov::op::PadMode pad_mode = ov::op::PadMode::CONSTANT;
     /// @brief Border value that is used in constant mode.
-    float pad_value = 0.0f;
+    float pad_value{0.0};
+    /// @brief Allow negative values in pads_begin and pad_end.
+    bool allow_negative_pad;
```

Comment:
Is it ok without default value?

Reply:
> Is it ok without default value?

It is ok since there are default value for ctor's argument. 
Though for sake of code consistency indeed it is better to have default value here. 
Fixed.